classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :@label, :@students
+
+  def initialize(label)
+    super()
```

Comment:
- [ ] I don't think there is any use for this `super()` definition, as Classroom class does not inherit from any class

Reply:
Thank you for your detailed explanation. You made it sound easy to understand. I wish I could get more lessons about this. Can you suggest anything for me? 
Thank you

Reply:
Hi @Bhekisile pardon the late reply. The free Ruby course on Codecademy is a good introduction to classes in Ruby, in my view. Regular practice also helps solidify new concepts.